src/cfnetwork.cs

```diff
@@ -67,6 +67,10 @@ interface CFHTTPMessage {
 		[Internal][Field ("kCFHTTPVersion2_0", "CFNetwork")]
 		NSString _HTTPVersion2_0 { get; }
 
+		[Mac (10, 16), iOS (14,0), TV (14, 0)]
+		[Internal][Field ("kCFHTTPVersion3_0", "CFNetwork")]
+		NSString _HTTPVersion3_0 { get; }
```

Comment:
use `IntPtr` since it's internal and you only use the `Handle` of the `NSString` anyway
you'll same some memory and remove null checks :)